### PR TITLE
Initial Release

### PR DESCRIPTION
**Pull request by kolkheang**
@mjhenkes 
@zplata 
@shriniketsarkar 
@kpshek

### CODE REVIEW

**Comment by poloka** [comment]
Is there a JIRA associated to this PR?

**Comment by jpruetting** [comment]
+1

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+6.0.0
```
[the remark this answers is not in the document]
Comment (mjhenkes) [reply .nvmrc]: +1 but lets make that change on a subsequent pull request.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* Fork the project.
+* Use a feature branch.
+* Write [good commit messages][3].
+* Use the same coding conventions as the rest of the project.
+* Commit locally and push to your fork until you are happy with your contribution.
+* Make sure to add tests and verify all the tests are passing when merging upstream.
+* Add an entry to the [Changelog][4] accordingly.
+* Please add your name to the CONTRIBUTORS.md file. Adding your name to the CONTRIBUTORS.md file signifies agreement to all rights and reservations provided by the [License][5].
+* [Squash related commits together][6].
+* Open a [pull request][7].
```
[the remark this answers is not in the document]
Comment (mjhenkes) [reply CONTRIBUTING.md]: +1 but lets make that change on a subsequent pull request.

**Pull request merged**